std/socket.d

```diff
@@ -2766,7 +2766,7 @@ public:
      * the connection to be made. If the socket is nonblocking, connect
      * returns immediately and the connection attempt is still in progress.
      */
-    void connect(Address to) @trusted
+    void connect(Address to) @trusted const
```

Comment:
Yes? It means that the object itself will not be modified.

A different question would be, does const on a method that mutates the state of what it is representing (OS socket) make sense?